core-ui/src/components/Predefined/List/Events.list.js

```diff
@@ -4,68 +4,128 @@ import {
   useGetList,
   ReadableCreationTimestamp,
 } from 'react-shared';
-import { useTranslation } from 'react-i18next';
-import { Link } from 'fundamental-react';
+import { useTranslation, Trans } from 'react-i18next';
 import {
-  useMessageList,
-  EVENT_MESSAGE_TYPE,
-  RESOURCE_PATH,
-} from 'hooks/useMessageList';
+  Link as DescriptionLink,
+  PageHeader,
+  prettifyNamePlural,
+} from 'react-shared';
+import { Icon, Link } from 'fundamental-react';
+import { useMessageList, EVENT_MESSAGE_TYPE } from 'hooks/useMessageList';
+
+export const EventsList = ({ ...otherParams }) => {
+  // TODO EVENTS DESCRIPTION
+  const description = (
+    <Trans i18nKey="events.description">
+      <DescriptionLink
+        className="fd-link"
+        url="https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/"
+      />
+    </Trans>
+  );
 
-export const EventsList = ({ i18n, ...otherParams }) => {
-  return <Events namespace={otherParams.namespace} i18n={i18n} />;
+  return (
+    <>
+      {!otherParams.isCompact && (
+        <PageHeader
+          title={prettifyNamePlural(
+            otherParams.resourceName,
+            otherParams.resourceType,
+          )}
+          description={description}
+        />
+      )}
+      <Events {...otherParams} />
+    </>
+  );
 };
 
-function Events({ namespace, i18n }) {
-  const { t } = useTranslation();
+export const Events = ({ ...otherParams }) => {
+  const { t, i18n } = useTranslation();
+  const { resourceUrl } = otherParams;
 
-  const url = `/api/v1/namespaces/${namespace}/events`;
-  const { loading = true, error, data: items } = useGetList()(url, {
-    pollingInterval: 3300,
-  });
+  const { loading = true, error, data: items } = useGetList(otherParams.filter)(
+    resourceUrl,
+    {
+      pollingInterval: 3300,
+    },
+  );
 
   const {
     displayType,
     sortedItems,
     messageSelector,
+    formatInvolvedObject,
+    formatSourceObject,
     navigateToObjectDetails,
   } = useMessageList(items);
 
-  const formatInvolvedObject = obj => {
-    const isLink = !!RESOURCE_PATH[obj.kind];
-    return isLink ? (
-      <Link className="fd-link" onClick={() => navigateToObjectDetails(obj)}>
-        {obj.name}
-      </Link>
-    ) : (
-      obj.name
-    );
-  };
-
   const entries =
     displayType.key === EVENT_MESSAGE_TYPE.ALL.key
       ? sortedItems
       : sortedItems.filter(e => e.type === displayType.key);
 
   const headerRenderer = () => [
-    t('namespaces.events.headers.message'),
-    t('namespaces.events.headers.object'),
-    t('namespaces.events.headers.object-type'),
-    t('namespaces.events.headers.created'),
+    t('common.labels.name'),
+    t('events.headers.type'),
+    t('events.headers.message'),
+    t('common.labels.namespace'),
+    t('events.headers.involved-object'),
+    t('events.headers.source'),
+    t('events.headers.count'),
```

Comment:
Maybe we could move this column to the details only?

Reply:
Together with Valentin, we decided to reduce the space needed for type columb (use icon not text) and to remove the namespace completely because the same information is in the involved object column